eth/tracers/native/call.go

```diff
@@ -130,10 +139,37 @@ func (t *callTracer) CaptureEnd(output []byte, gasUsed uint64, _ time.Duration,
 
 // CaptureState implements the EVMLogger interface to trace a single step of VM execution.
 func (t *callTracer) CaptureState(pc uint64, op vm.OpCode, gas, cost uint64, scope *vm.ScopeContext, rData []byte, depth int, err error) {
+	if !t.config.WithLog {
+		return
+	}
+	switch op {
```

Comment:
This will probably be a huge performance hit (for those who need logs). We could alternatively add a `CaptureLog` hook. Good thing about it is adding it won't be polluting the main EVM loop, only one instruction. @holiman Wdyt?